ui/appui-react/src/appui-react/statusfields/SelectionScope.tsx

```diff
@@ -55,9 +55,9 @@ function SelectionScopeFieldComponent(props: SelectionScopeFieldProps) {
       )}
       style={props.style}
     >
-      <label className="uifw-statusFields-selectionScope-label">
+      <Label className="uifw-statusFields-selectionScope-label">
```

Comment:
@FlyersPh9  I also noticed something about iTwinUI labels when working on this item. It looks like label text and text inside the input is not vertically aligned. For example, in the screenshot I used `LabeledInput` without any additional styling (except I disabled the semi-bold font so it's easier to compare). So both label and input are from iTwinUI and they're wrapped together by iTwinUI (although the same thing happens if I use `Label` and `Input` separately). You can see that the label is slightly lower than the text inside the input. This is also the case in [Ladle docs](https://itwin.github.io/iTwinUI/react/?story=label--inline). However, in [these docs](https://itwinui.bentley.com/docs/input#inline) the label and input text seems to align properly. Could you take a look?

P.S. it aligns if I wrap the `LabeledInput` in a div with `display:flex` and odd height e.g. `height: 35px`.

![image](https://github.com/iTwin/appui/assets/138560995/8cf2dec0-579e-4bf3-bbb3-e56f0f1d1926)